docker-compose.yml

```diff
@@ -12,11 +12,22 @@ services:
       - POSTGRES_USER=jonas
       - POSTGRES_HOST=db
       - POSTGRES_PORT=5432
+      - POSTGRES_DB=jonas
   point_of_sale:
-    restart: always
```

Comment:
put this back and set it as `on-failure`

Reply:
Ok